curriculum/challenges/english/24-b1-english-for-developers/learn-about-speculation-and-requests/67f519febaeacd154eb0f9f1.md

```diff
@@ -0,0 +1,82 @@
+---
+id: 67f519febaeacd154eb0f9f1
+title: Task 96
+challengeType: 22
+dashedName: task-96
+---
+
+<!-- (audio) Jake: For now, let's gather more information and stay vigilant. -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`For now, let's gather more information and stay BLANK.`
+
+## --blanks--
+
+`vigilant`
+
+### --feedback--
+
+This word means being alert and watching carefully for potential dangers or problems.
+
+# --explanation--
+
+`Vigilant` means being watchful and cautious to detect possible threats. For example:
+
+- `The IT team is always vigilant when monitoring the network.` - The IT team stays alert and careful while checking the network.
+
+- `You should be vigilant when clicking on unknown links in emails.` - You need to be careful and pay attention to suspicious email links.
```

Comment:
I don't know how to explain `you` in the example, but it should be modified according to the standard doc.